pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "typing-extensions >= 4.5.0, < 5",
-  "frequenz-api-common >= 0.5.4, < 6",
+  "frequenz-api-common >= 0.6.1, < 7",
```

Comment:
I think we don't need to bump the min version, that way others can still use the older dependency until they need it

Reply:
Sounds good! I will change it accordingly.

Reply:
Speaking for reporting, we would need v0.6.x for the new names (no need to have a branch that's compatible with v0.5.x in our case).

Reply:
Let's just do that on demand, no need to add extra work without cause